doc/user-guide/using-python-api.rst

```diff
@@ -0,0 +1,37 @@
+.. _Using-Python-In-Docs
+Using the Python API in the docs
+===============================
```

Comment:
this looks like a document that we want to embed whenever we display the tabs right? I'd say let's move it to the FAQ instead of being a complete section.

Also, I think it'd be good to automatically embed this at the bottom of the Python API tab so we don't have to put this there every time